Firestore/Example/Tests/Local/FSTQueryCacheTests.m

```diff
@@ -56,7 +56,8 @@ - (void)testReadQueryNotInCache {
 - (void)testSetAndReadAQuery {
   if ([self isTestBaseClass]) return;
 
-  FSTQueryData *queryData = [self queryDataWithQuery:_queryRooms targetID:1 version:1];
+  FSTQueryData *queryData =
```

Comment:
It seems like most of these tests don't care about the targetID, the listenSequenceNumber, or the version. Could you add a queryDataWithQuery: overload that defaults the targetID, listenSequenceNumber, and version for tests that don't care to cut down on the noise in here?

Alternatively, since it seems like the listenSequenceNumber-testing methods don't actually use this helper maybe we can have queryDataWithQuery just default the listen sequence number?

Reply:
I added a helper that just takes a query, and transitioned most of the helper calls over to it.